draft-ietf-quic-transport.md

```diff
@@ -1831,7 +1831,9 @@ Connection ID:
 Receivers send ACK frames to inform senders which packets they have received and
 processed, as well as which packets are considered missing.  The ACK frame
 contains between 1 and 256 ACK blocks.  ACK blocks are ranges of acknowledged
-packets.
+packets. Implementations SHOULD NOT generate ACK packets in response to packets
```

Comment:
I'd go for MUST NOT and MUST ACK, though I admit it's a bit difficult to enforce either.